ROADMAP.md

```diff
@@ -1,74 +1,121 @@
+- IoT Thing Group 
+- Lambda Access Points for S3
+- ECR Public Repositories
 
-We also launched a fully generated provider, the AWS Cloud Control (AWSCC) provider for Terraform. The AWSCC provider is currently in Technical Preview. Please check it out and let us know what you think.
+From February ‘22 - April ‘22, we will be prioritizing the following areas of work:
 
-- [HashiCorp Blog Announcement](https://www.hashicorp.com/blog/announcing-terraform-aws-cloud-control-provider-tech-preview)
-- [GitHub Repository](https://github.com/hashicorp/terraform-provider-awscc)
-- [AWS Cloud Control on the Terraform Registry](https://registry.terraform.io/providers/hashicorp/awscc/latest)
+## New Services  
 
-From November ‘21- January ‘22, we will be prioritizing the following areas of work:
+### AWS AppFlow
 
-## Provider Version v4.0.0
+Issue: [#11145](https://github.com/hashicorp/terraform-provider-aws/issues/11145)
 
-Issue: [#20433](https://github.com/hashicorp/terraform-provider-aws/issues/20433)
+_[Amazon AppFlow](https://aws.amazon.com/appflow/) is a fully managed integration service that enables you to securely transfer data between Software-as-a-Service (SaaS) applications_
 
-The next major release of the provider will include the adoption of the AWS Go SDK v2.0 as well as a refactor of one of our oldest and most used resources: S3.
+Support for Amazon AppFlow will include:
 
-There will also be the usual deprecations and sometimes breaking changes to existing resources which are necessary to maintain consistency of behavior across resources. Our goal is to focus on standardization to reduce technical debt and lay a strong foundation for future enhancement initiatives within the provider.
+New Resource(s):
 
-For details of the changes in full please refer to #20433. We would love to hear your feedback.
+- `aws_appflow_flow`
+- `aws_appflow_connector_profile`
+
+
+### Amazon Global Networks
+
+Issue: [#11132](https://github.com/hashicorp/terraform-provider-aws/issues/11132)
+
+_A [Global Network](https://docs.aws.amazon.com/vpc/latest/tgwnm/global-networks.html) is a container for your network objects. After you create it, you can register your transit gateways and define your on-premises networks in the global network._
+
+Support for Global Networks will include:
+
+New Resource(s):
+
+- aws_networkmanager_global_network
+- aws_networkmanager_site
+- aws_networkmanager_link
+- aws_networkmanager_device
+- aws_networkmanager_transit_gateway_registration
+
+
+### Amazon OpenSearch Service
+
+Issue: [#20853](https://github.com/hashicorp/terraform-provider-aws/issues/20853)
+
+_[Amazon OpenSearch Service](https://aws.amazon.com/opensearch-service/) is a distributed, open-source search and analytics suite used for a broad set of use cases like real-time application monitoring, log analytics, and website search_
+
+Support for Amazon OpenSearch Service will include:
+
+New Resource(s):
+
+- `aws_elasticsearch_domain`
```

Comment:
Confirmation on this set of resources @breathingdust

Reply:
Might leave this out, ideally we will be able to use the same resources but we need to dig in a little further